soccer/src/soccer/kick_evaluator.cpp

```diff
@@ -102,8 +106,8 @@ KickResults KickEvaluator::eval_pt_to_seg(Point origin, Segment target) {
     // Create function with only 1 input
     // Rest are bound to constant values
     function<tuple<float, float>(float)> ke_func =
-        bind(&eval_calculation, std::placeholders::_1, (kick_mean->value()),
-             (kick_std_dev->value()), cref(bot_means), cref(bot_st_devs), cref(bot_vert_scales),
+        bind(&eval_calculation, std::placeholders::_1, kick_evaluator::PARAM_kick_mean,
+             kick_evaluator::PARAM_kick_std_dev, cref(bot_means), cref(bot_st_devs), cref(bot_vert_scales),
              target_width / -2, target_width / 2);
```

Comment:
```suggestion
        [] (float x) {
            return eval_calculation(x,
                                    kick_evaluator::PARAM_kick_mean,
                                    kick_evaluator::PARAM_kick_std_dev,
                                    cref(bot_means),
                                    cref(bot_st_devs),
                                    cref(bot_vert_scales),
                                    -target_width / 2,
                                    target_width / 2);
```

Reply:
This code raised an error, so I added & inside the bracket(not sure that's the right way to fix it).